config/software/openssl-windows.rb

```diff
@@ -36,6 +36,10 @@
 version('1.0.0n') { source md5: "9506530353f3b984680ec27b7270874a" }
 version('1.0.0q') { source md5: "577dbe528415c6f178a9431fd0554df4" }
 version('1.0.0r') { source md5: "25402ddce541aa54eb5e114721926e72" }
+version('1.0.1m') {
+  source url: "https://github.com/jdmundrawala/knapsack-recipes/releases/download/openssl-1.0.1m/openssl-1.0.1m-x86-windows.tar.lzma",
```

Comment:
@sethvargo @jdmundrawala I think this is OK for the moment. Once @jdmundrawala gets his fix in the upstream project the software definition will just pull from there like all the other projects.